.springBeans

```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Let's separate them - that way I can more easily merge one of them. 
Cheers, 
Eugen.